src/app/components/challenge/challengesettings/challengesettings.component.ts

```diff
@@ -63,7 +68,8 @@ export class ChallengesettingsComponent implements OnInit {
   }
 
   updateView() {
-    this.bannedEmailIds = this.challenge.banned_email_ids;
+    this.bannedEmailIds = this.challenge.banned_email_ids || [];
+    this.formerBannedEmailIds = this.bannedEmailIds.concat(); // Creating deep copy
```

Comment:
Can you please remove the comment from here and from other place also?

Reply:
But isn't this line confusing without the comment?

Reply:
okay then lets keep it to avoid confusion :smile: